docker-app/qfieldcloud/core/tests/test_deleteorphanedfiles.py

```diff
@@ -6,9 +6,13 @@
 from qfieldcloud.core.utils import get_project_files_count, get_s3_bucket
 from qfieldcloud.core.utils2 import storage
 
+from .utils import setup_subscription_plans
 
-class NormalizeBookTitlesTests(TestCase):
+
+class QfcTestCase(TestCase):
     def setUp(self):
+        setup_subscription_plans()
```

Comment:
not clear how this relates to the rest ? is it intended ?

Reply:
I wonder how it worked before, tbh. If we don't call this function, then the next line `self.u1 = Person.objects.create(username="u1")` is failing.